nodl_python/nodl/errors.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2020 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the
+# GNU Limited General Public License version 3, as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranties of MERCHANTABILITY, SATISFACTORY QUALITY, or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Limited General Public License for more details.
+#
+# You should have received a copy of the GNU Limited General Public License along with
+# this program. If not, see <http://www.gnu.org/licenses/>.
+
+from typing import TYPE_CHECKING
+
+from lxml import etree
+
+if TYPE_CHECKING:  # pragma: no cover
+    import rclpy.qos
+
+
+class NoDLError(Exception):
+    """Base class for all NoDL exceptions."""
+
+    pass
```

Comment:
I don't believe you need this... do you?

Reply:
87c051f